landlab/components/hand_calculator/hand_calculator.py

```diff
@@ -0,0 +1,195 @@
+# -*- coding: utf-8 -*-
+"""Landlab component to calculate height above nearest drainage."""
+from warnings import warn
+
+import numpy as np
+
+from landlab import Component
+
+
+class HeightAboveDrainage(Component):
+    """
+    Calculate the elevation difference between each node and its nearest
+    drainage node in a DEM.
+
+    This component implements the method described by Nobre et al (2011). A
+    single direction flow director (D8 or steepest descent) must be run prior
+    to HeightAboveDrainage to supply the flow directions. This component does
+    not fill depressions in a DEM, instead it treats them as drainage nodes.
```

Comment:
"DEM, but rather treats them..." or "DEM; instead, it treats them..."

Reply:
Changed this as well, thanks!